service/history/historyEngine_test.go

```diff
@@ -187,8 +187,8 @@ func (s *engineSuite) TestGetMutableStateSync() {
 	ctx := context.Background()
 
 	workflowExecution := types.WorkflowExecution{
-		WorkflowID: common.StringPtr("test-get-workflow-execution-event-id"),
-		RunID:      common.StringPtr(constants.TestRunID),
+		WorkflowID: "test-get-workflow-execution-event-id",
+		RunID:      constants.TestRunID,
 	}
```

Comment:
> 528  service/history/historyEngine_test.go 

bleh.
I wonder if it'd be worth cleaning up the extreme duplication in so many of these test files... maybe after all the type changing is done?